Dockerfile

```diff
@@ -1,66 +1,37 @@
-FROM golang:1.11 as builder
+FROM golang:1.13.1 as builder
 
-# Set up workdir
+# Set up workdir to make go mod work
 WORKDIR /cayley
 
-# Restore dependencies
+# Install dependencies first
 COPY go.mod go.sum ./
 RUN go mod download
 
-# This will be used to init cayley and as config file in the final image.
-# Make sure you start every path with %PREFIX% to make it available in both 
-# the builder image and the final image.
-RUN echo '{"store":{"backend":"bolt","address":"%PREFIX%/data/cayley.db"}}' > config.json
```

Comment:
I don't think we should change the default configuration for Docker.

First, it's a breaking change.

Second, I think you usually expect a persistence from services running in Docker, so things like:
```
docker stop cayley
docker start cayley
``` 
shouldn't drop your data.

Yes, it's inconsistent with CLI, but CLI use case is different - it's mostly for interactive usage. So you can quickly load any nquads file to memory, play with it, and setup a proper config with persistence if needed.

So instead of changing the default behavour for either CLI or Docker I think we should document it better.

Reply:
I agree. So I think we should streamline the behaviour the other way in the next breaking change. Similarly to other database persistency should be the default.